docs/products/postgresql/howto/check-avoid-transaction-id-wraparound.rst

```diff
@@ -0,0 +1,21 @@
+Check and avoid transaction ID wraparound
+=========================================
+
+The PostgreSQL® transaction control mechanism assigns a transaction ID to every row that is modified in the database; these IDs control the visibility of that row to other concurrent transactions.
+
+The transaction ID is a 32-bit number, where 2 billion (2 thousand million) IDs are always in the "visible past" and the remainder (about 2.2 billion) are reserved for future transactions and not visible to the running transaction. To avoid a transaction wraparound and having old, existing rows invisible when more transactions are created, PostgreSQL® requires an occasional cleanup and "freezing" of old rows. 
+
```

Comment:
What about adding subtitles for the actions? Like here "Execute `VACUUM` manually"

This way we are cutting the wall of text in pieces and people can easily spot the part they're interested into

Reply:
Yes, that seems like a good idea - adding more navigation seems useful.
I tried a ``rubric`` directive, but plain titles seems better.
Went with "Automatic or manual vacuuming" here, as a first attempt.